pkg/resource/deploy/deployment_executor.go

```diff
@@ -524,19 +524,19 @@ func (ex *deploymentExecutor) handleSingleEvent(event SourceEvent) error {
 	var steps []Step
 	var err error
 	switch e := event.(type) {
-	case ContinueResourceRefreshEvent:
-		logging.V(4).Infof("deploymentExecutor.handleSingleEvent(...): received ContinueResourceRefreshEvent")
+	case ContinueResourceImportEvent:
+		logging.V(4).Infof("deploymentExecutor.handleSingleEvent(...): received ContinueResourceImportEvent")
```

Comment:
Had to flip these around because ContinueResourceRefreshEvent now matches the interface for ContinueResourceImportEvent so Import was never hit if it came second.